RecoParticleFlow/PFTracking/python/trackerDrivenElectronSeeds_cfi.py

```diff
@@ -68,3 +68,6 @@
 
 from Configuration.ProcessModifiers.egamma_lowPt_exclusive_cff import egamma_lowPt_exclusive
 egamma_lowPt_exclusive.toModify(trackerDrivenElectronSeeds,MinPt = 1.0)
+
+from Configuration.ProcessModifiers.seedingDeepCore_cff import seedingDeepCore
+seedingDeepCore.toModify(trackerDrivenElectronSeeds, MaxPt = 50 )
```

Comment:
I'm confused, why do you apply this change ?

Reply:
it was an old feature, actually not useful (the default MaxPt=50 yet). Removed in the last commit